wwwapp/auth.py

```diff
@@ -35,7 +33,6 @@ def login_view(request):
         else:
             client = access.api_client
             user_info = client.get_profile_info(raw_token=access.access_token)
-            context['info'] = user_info
```

Comment:
Before you ask - yes, this line seems to be an unused leftover from debugging during development (see 3cb0a1528627186c4987874e98cf71e7f502c4b5, it was never used in the first place)